src/utils/map-state.js

```diff
@@ -2,16 +2,16 @@ import {PerspectiveMercatorViewport} from 'viewport-mercator-project';
 import assert from 'assert';
 
 // MAPBOX LIMITS
-const MAX_PITCH = 60;
-const MAX_ZOOM = 20;
+export const MAPBOX_MAX_PITCH = 60;
+export const MAPBOX_MAX_ZOOM = 20;
```

Comment:
These are limitations of Mapbox. These two numbers are only ever used as default prop values.